### ISSUE TITLE
Presence Good, Status not so much...

### ISSUE THREAD

**Issue by incompletebiped** [paultyng/streamdeck-slack#7]
I have really been need the Slack Presence and that is working great but I can't get the Status to work. It's funny, I have the Slack plugin by Elgato that works great for Direct Messages (with User Status) and to Open Channels. Then I have Slack Status by Ellreka that does work for Update Slack Status and Clear Slack Status but doesn't do anything else. Then I got your plugin for the Slack Presence. I was hoping yours would replace them all but not quite (yet). Here's hoping you are still working on it. 

It seems to have a problem retaining the API for the Slack Status.

**Comment by incompletebiped** [comment]
Update: the Slack Presence does not seem to be retaining the API Token either. It will stay in the field for a little while and it works during that time. Everything was working when I ended my day yesterday at 10pm but by 8am this morning toggling the buttons no longer did anything. The color indicator would change on the Stream Deck but no effect in Slack. When I went to the settings it said the API Token was undefined. I put in the token again and they worked. I'll try to time how long before the token disappears.
